### PR TITLE
Diabetes_Prediction

### PR DESCRIPTION
**Pull request by amoghatsunil**
jupyter notebook link:
https://colab.research.google.com/drive/1Op49nQIa-nJn42Ibczznv5IKMIlyhBXy?usp=sharing

KNN:
train accuracy 0.8206106870229007
auc 0.6750868751670676
test accuracy  0.7461538461538462
Precision  0.6666666666666666
Recall  0.46511627906976744
F1 Score  0.547945205479452
[0.81132075 0.81132075 0.69230769 0.80769231 0.80769231]
0.786066763425254

LOGISTIC REGRESSION:
Training set accuracy: 0.802
Test set accuracy: 0.723
train accuracy 0.8015267175572519
auc 0.6750868751670676
test accuracy  0.7461538461538462
Precision  0.6666666666666666
Recall  0.46511627906976744
F1 Score  0.547945205479452
[0.83018868 0.75471698 0.73076923 0.80769231 0.84615385]
0.7939042089985487

DECISION TREE
Accuracy on training set: 0.851
Accuracy on test set: 0.762
train accuracy 0.8015267175572519
auc 0.6750868751670676
test accuracy  0.7461538461538462
Precision  0.6666666666666666
Recall  0.46511627906976744
F1 Score  0.547945205479452
[0.83018868 0.75471698 0.73076923 0.80769231 0.84615385]
0.7939042089985487

SVM :
Accuracy on training set: 0.824
Accuracy on test set: 0.715
auc 0.628575247260091
Precision  0.6153846153846154
Recall  0.37209302325581395
F1 Score  0.463768115942029

SVM Cross_val:
train_accuracy: 0.7935897435897437
Test accuracy: 0.738
{'clf__C': 1000.0, 'clf__gamma': 0.001, 'clf__kernel': 'rbf'}

### CODE REVIEW

**Comment by smaranjitghose** [comment]
> _No description provided._

Please checkout the guidelines for PR, I gave in the issues
I was expecting a link to preview the Jupyter Notebook in Google Colab and more description on the PR, about your findings, performance metrics used, what did you use to decide the best model, different algos used

**Comment by amoghatsunil** [comment]
@smaranjitghose 
i have added link to the notebook in google colab

**Comment by smaranjitghose** [comment]
> @smaranjitghose
> i have added link to the notebook in google colab

Please only give the best model weights in the PR, the rest can be inferred from the notebook

File: diabetes_pred/Models/readme [hunk 1]
```diff
@@ -0,0 +1 @@
+
```
Comment (smaranjitghose) [inline diabetes_pred/Models/readme:1]: Not Required
The notebook is self explanatory

File: diabetes_pred/readme [hunk 2]
```diff
@@ -0,0 +1,49 @@
+**Diabetes_Prediction**
+#5
+KNN:
+train accuracy 0.8206106870229007
+auc 0.6750868751670676
```
Comment (smaranjitghose) [inline diabetes_pred/readme:5]: Not Required! The notebook is self explanatory

File: diabetes_pred/readme [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+Accuracy on test set: 0.715
+auc 0.628575247260091
+Precision  0.6153846153846154
+Recall  0.37209302325581395
+F1 Score  0.463768115942029
+
+SVM Cross_val:
+train_accuracy: 0.7935897435897437
+Test accuracy: 0.738
+{'clf__C': 1000.0, 'clf__gamma': 0.001, 'clf__kernel': 'rbf'}
```
Comment (smaranjitghose) [inline diabetes_pred/readme:49]: Always give the comparative analysis in a table

**Review by smaranjitghose — CHANGES_REQUESTED** [review]
